config/webpack.config.prod.js

```diff
@@ -318,8 +318,5 @@ module.exports = {
     tls: 'empty',
     child_process: 'empty',
   },
-  performance: {
-    maxAssetSize: 6000000,
-    maxEntrypointSize: 6000000,
-  },
+  performance: { hints: false },
```

Comment:
Again with this one: I got an error on the previous commit that the combined bundle size was over 5mb so I disabled this, only to find on the next run that the bundle size was way less. 

What is going on?